src/components/MyAccount/RequestsTab/UpdateLocation.tsx

```diff
@@ -15,28 +15,32 @@ import {
 } from "@nypl/design-system-react-components"
 import type { Hold, SierraCodeName } from "../../../types/myAccountTypes"
 import styles from "../../../../styles/components/MyAccount.module.scss"
-import { useContext, useRef, useState, useEffect } from "react"
+import { useContext, useRef, useState, useEffect, type Dispatch } from "react"
 import { BASE_URL } from "../../../config/constants"
 import { PatronDataContext } from "../../../context/PatronDataContext"
 
 interface UpdateLocationPropsType {
   hold: Hold
   pickupLocationOptions: SierraCodeName[]
-  key: number
+  key: string
   patronId: number
+  setLastUpdatedHoldId: Dispatch<string>
+  focus: boolean
```

Comment:
How do you feel about updating this to `isFocused`? Makes it slightly more readable.

Reply:
Ended up refactoring this to match the pattern I'm using with the loading states.